pkg/ccl/streamingccl/streamingest/alter_replication_job.go

```diff
@@ -161,7 +161,7 @@ func alterReplicationJobHook(
 			return err
 		}
 		if tenInfo.TenantReplicationJobID == 0 {
-			return errors.Newf("tenant %q does not have an active replication job", tenantName)
+			return errors.Newf("tenant %q is not a replicated tenant", tenantName)
```

Comment:
hmm is this change necessary? Tenant foo could have been created as a result of c2c replication, so it could be confusing to say that it is not a replicated tenant if the job ID is 0. IMO it is sufficient to say that there is no active replication job associated with this tenant and so the replication information you are trying to access is unavailable.